examples/arithmetic/Cargo.toml

```diff
@@ -11,7 +11,7 @@ name = "uniffi_arithmetic"
 
 [dependencies]
 uniffi_macros = {path = "../../uniffi_macros"}
-uniffi = {path = "../../uniffi"}
+uniffi = {path = "../../uniffi", features=["builtin-bindgen"]}
```

Comment:
I wonder if we should add a comment there (or at the feature declaration in `uniffi` and `uniffi_build`) to say that you're not supposed to use this?

Reply:
Yes, good idea. I tried to explain it a bit in the code where it's used, but consumers are not likely to look there.